src/v2/view-builder/views/ov/EnrollementChannelDataOktaVerifyView.js

```diff
@@ -122,8 +122,6 @@ export default BaseAuthenticatorView.extend({
     });
   },
   postRender() {
-    if (!this.settings.get('features.sameDeviceOVEnrollmentEnabled')) {
-      this.add(SwitchEnrollChannelLinkView, 'form');
-    }
```

Comment:
Since the conditional is gone, what is the impact of this change?

Reply:
now with this change the switch channel link will display on the enrollment channel page (sms, email etc. ) when the 
sameDeviceOVEnrollmentEnabled feature is enabled.
<img width="488" alt="Screenshot 2024-07-18 at 3 54 11 PM" src="https://github.com/user-attachments/assets/ee014218-ea7a-48da-bc51-8cab8a648c7f">

which matches the figma here: https://www.figma.com/file/ZP8O9yyMXIsnOhyt8yOKcw/OV-local-enrollment-enhancements?type=design&node-id=2728-15464&mode=design&t=mZ4a71AGYREvZH2Q-0